src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -95,232 +32,19 @@ const NavBar: React.FC<NavBarPropsType> = ({
       : undefined
   );
 
-  const {
-    location: { pathname },
-  } = useHistory();
-  const isOnPlaya = pathname.toLowerCase() === venueInsideUrl(PLAYA_VENUE_ID);
-
-  const now = firebase.firestore.Timestamp.fromDate(new Date());
-  const futureUpcoming =
-    venue?.events?.filter((e) => e.ts_utc.valueOf() > now.valueOf()) ?? []; //@debt typing does this exist?
-
-  const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
-
-  const hasRadioStations = radioStations && radioStations.length;
-  const isSoundCloud =
-    !!hasRadioStations && RegExp("soundcloud").test(radioStations![0]);
-
-  const sound = useMemo(
-    () =>
-      radioStations && hasElements(radioStations) && !isSoundCloud
-        ? new Audio(radioStations[0])
-        : undefined,
-    [isSoundCloud, radioStations]
-  );
-
-  const [isRadioPlaying, setIsRadioPlaying] = useState(false);
-  const { volume, setVolume } = useRadio(isRadioPlaying, sound);
-
-  const radioFirstPlayStateLoaded = useRef(false);
-  const showRadioOverlay = useMemo(() => {
-    if (!radioFirstPlayStateLoaded.current) {
-      const radioFirstPlayStorageKey = "radioFirstPlay";
-      const radioFirstPlayState = localStorage.getItem(
-        radioFirstPlayStorageKey
-      );
-      if (!radioFirstPlayState) {
-        localStorage.setItem(radioFirstPlayStorageKey, JSON.stringify(true));
-        return true;
-      }
-      radioFirstPlayStateLoaded.current = true;
-    }
-    return false;
-  }, [radioFirstPlayStateLoaded]);
-
-  const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
-  const toggleEventSchedule = useCallback(() => {
-    setEventScheduleVisible(!isEventScheduleVisible);
-  }, [isEventScheduleVisible]);
-  const hideEventSchedule = useCallback(() => {
-    setEventScheduleVisible(false);
-  }, []);
-
-  const parentVenueId = venue?.parentId ?? "";
-  const backToParentVenue = useCallback(() => {
-    window.location.href = venueInsideUrl(parentVenueId);
-  }, [parentVenueId]);
-
-  const navigateToHomepage = useCallback(() => {
-    const venueLink =
-      redirectionUrl ?? venueId ? venueInsideUrl(venueId ?? "") : "/";
-
-    window.location.href = venueLink;
-  }, [redirectionUrl, venueId]);
-
-  const handleRadioEnable = useCallback(() => setIsRadioPlaying(true), []);
-
-  const [showRadioPopover, setShowRadioPopover] = useState(false);
-
-  const toggleShowRadioPopover = useCallback(
-    () => setShowRadioPopover((prevState) => !prevState),
-    []
-  );
-
-  if (!venueId || !venue) return null;
-
-  // TODO: ideally this would find the top most parent of parents and use those details
-  const navbarTitle = parentVenue?.name ?? venue.name;
```

Comment:
[note] Moved to `src/components/molecules/NavbarSchedule/NavbarSchedule.tsx`

Reply:
Also, the code in the new location no longer has this `TODO` comment attached, which I believe it should still have.